### PR TITLE
Update schema with relationship between users and orders

### PR DESCRIPTION
**Pull request by lrs8810**
- Add one to many relationship between user and orders
- All tests passing

### CODE REVIEW

File: db/migrate/20190824200120_create_orders.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@ def change
       t.string :city
       t.string :state
       t.integer :zip
+      t.integer :status, default: 0
```
Comment (freeheeling) [inline db/migrate/20190824200120_create_orders.rb:11]: Add column from the command line: rails g migration AddStatusToOrders status:integer
I believe a default value must still be added in the migration file, however.

File: spec/features/merchants/statistics_spec.rb [hunk 2]
```diff
@@ -17,10 +17,18 @@
       @tire = @meg.items.create(name: 'Gatorskins', description: "They'll never pop!", price: 100, image: 'https://www.rei.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588', inventory: 12)
       @pull_toy = @brian.items.create(name: 'Pull Toy', description: 'Great pull toy!', price: 10, image: 'http://lovencaretoys.com/image/cache/dog/tug-toy-dog-pull-9010_2-800x800.jpg', inventory: 32)
       @dog_bone = @brian.items.create(name: 'Dog Bone', description: "They'll love it!", price: 20, image: 'https://img.chewy.com/is/image/catalog/54226_MAIN._AC_SL1500_V1534449573_.jpg', active?: false, inventory: 21)
-
-      @order_1 = Order.create!(name: 'Meg', address: '123 Stang Ave', city: 'Hershey', state: 'PA', zip: 17_033)
-      @order_2 = Order.create!(name: 'Brian', address: '123 Zanti St', city: 'Denver', state: 'CO', zip: 80_204)
-      @order_3 = Order.create!(name: 'Mike', address: '123 Dao St', city: 'Denver', state: 'CO', zip: 80_210)
+      @user = User.create(
```
Comment (freeheeling) [inline spec/features/merchants/statistics_spec.rb:20]: There seem to be a lot of unnecessary instance variables used throughout this test file.

**Review by freeheeling — COMMENTED** [review]
No changes needed before merging.

**Review by cjkelling — APPROVED** [review]
This looks good. Great job adding a new migration that will connect our users and their orders. I appreciate that you went back and edited the tests to reflect the new relationship.

**Pull request merged**